DeriSock.ConsoleTest/.env.example

```
@@ -0,0 +1,2 @@
+DERIBIT_API_KEY=
+DERIBIT_API_SECRET=
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)